BaseFilter/sections/specific.txt

```diff
@@ -17182,6 +17182,7 @@ fijivillage.com#?#.contentleft > p > small:contains(/^ADVERTISEMENT$/)
 biertijd.com,biertijd.xxx#?##wrapper > div#sidebarcontainer > h2:contains(/Advertenties|Sponsors|Aanrader/)
 mumbaimirror.indiatimes.com#?##content > div[class*=" "]:has(> div > div > a[onclick])
 mumbaimirror.indiatimes.com#?#.latestCmmtData > .mrtoday > div[class*=" "]:has(> div > div > a[onclick])
+contexto.me#@#.share-btn
```

Comment:
The rule is ok, but wrong filter. Please move to Social filter to `allowlist.txt` like another rules there.